src/confitems.c

```diff
@@ -136,6 +136,8 @@ confitem_parse_sloppiness(const char *str, void *result, char **errmsg)
 			*value |= SLOPPY_PCH_DEFINES;
 		} else if (str_eq(word, "time_macros")) {
 			*value |= SLOPPY_TIME_MACROS;
+		} else if (str_eq(word, "no_clang_index_store")) {
```

Comment:
Except for `no_system_headers`, which I regard as a mistake (see #337), previous sloppiness names can be substituted for X in "ccache should be sloppy about X", so I think that it should be called `clang_index_store` without the `no_` prefix.

Also, the new value needs to be documented in `doc/MANUAL.adoc`. Maybe something like this?

```
*clang_index_store*::
    Ignore the Clang compiler option *-index-store-path* and its argument when
    computing the manifest hash. This is useful if you use Xcode, which uses an
    index store path derived from the local project path. Note that the index
    store won't be updated correctly on cache hits if you enable this option.
```